src/AdminConsole/Services/PasswordlessManagement/IPasswordlessManagementClient.cs

```diff
@@ -20,4 +20,6 @@ public interface IPasswordlessManagementClient
     Task EnabledManuallyGeneratedTokensAsync(string appId, string performedBy);
     Task DisabledManuallyGeneratedTokensAsync(string appId, string performedBy);
     Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request);
+    Task EnableMagicLinksAsync(string appId, string performedBy);
+    Task DisableMagicLinksAsync(string appId, string performedBy);
```

Comment:
Why don't we expose these as `EnableMagicLinksAsync(string appId, string performedBy, bool isEnabled)` instead, and call different endpoints internally?

Reply:
There's another [PR](https://github.com/bitwarden/passwordless-server/pull/360) that will make these obsolete if we choose to go with it.